scripts/globals/abilities/pets/crimson_howl.lua

```diff
@@ -13,8 +14,10 @@ function onAbilityCheck(player, target, ability)
 end;
 
 function onPetAbility(target, pet, skill, summoner)
-	local duration = 30 + summoner:getMod(MOD_SUMMONING);
-	target:addStatusEffect(EFFECT_WARCRY,9,0,duration);
-	skill:setMsg(MSG_BUFF);
-	return EFFECT_WARCRY;
+    local bonusTime = utils.clamp(summoner:getSkillLevel(SUMMONING_SKILL) - 300, 0, 200);

```

Comment:
I suggest using the ones in status.lua for now: https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/status.lua#L1889 since that global is already required here anyway.. For some derptarded reason different editors variable-lized some skill ID's multiple times in different globals without noticing each other's work. I plan to eventually make a pull request standardizing them all into a single place and single naming convention and I'll adjust all scripts using the existing ones at that time.